packages/tocco-ui/src/SignalBox/StyledSignalBox.js

```diff
@@ -0,0 +1,55 @@
+import {theme} from 'styled-system'
+import styled from 'styled-components'
+
+import {StyledSignalListItem} from '../SignalList'
+
+import {
+  oneOfPropTypeAndCompletelyMapped,
+  stylingCondition
+} from '../utilStyles'
+
+const ALLOWED_CONDITION = [
+  stylingCondition.BASE,
+  stylingCondition.DANGER,
+  stylingCondition.SUCCESS,
+  stylingCondition.WARNING
+]
+
+const COLORS = {
+  [stylingCondition.DANGER]: theme('colors.signal.dangerBg'),
+  [stylingCondition.SUCCESS]: theme('colors.signal.successBg'),
+  [stylingCondition.WARNING]: theme('colors.signal.warningBg'),
+  [stylingCondition.BASE]: theme('colors.base.fill.1')
+}
+
+const getColor = props => {
+  const color = COLORS[props.condition]
+  return color || theme('colors.base.fill.1')
+}
+
+// TODO icon should have line-height 1.4
```

Comment:
can we remove this comment?

Reply:
Yes, this will be fixed by commit 4b15dbd9dc9356b5b4a38b2089f89f02adf84a19 https://github.com/tocco/tocco-client/pull/460